.ci/install_kata_image.sh

```diff
@@ -70,8 +71,22 @@ build_rust_image() {
 	popd
 }
 
+build_image_for_cc () {
+	[ "${TEST_INITRD}" != "yes" ] || \
+		die "Build of initrd image for Confidential Containers is still unsupported."
```

Comment:
@wainersm, this is causing the SEV CI to fail.  What's the best approach to take here?

Reply:
I will have a look. With SEV we use an initrd instead of a ramdisk image (I was told the reason but I keeps forgetting, I am getting old).

Reply:
ok, @fidencio , you should remove that test & die from the code.